src/component/pages/Login.jsx

```diff
@@ -0,0 +1,101 @@
+import { useState, useEffect } from "react";
+
+function Login({ authenticatedUser, setAuthenticatedUser }) {
+  const [user, setUser] = useState({
+    first_name: "",
+    last_name: "",
+    email: "",
+    password: "",
+    street: "",
+    postcode: "",
+    phone: "",
+  });
+
+  console.log({ user, authenticatedUser });
+
+  useEffect(() => {
+    const userAsString = localStorage.getItem("user");
+
+    if (userAsString) {
+      const user = JSON.parse(userAsString);
```

Comment:
You can't use `const user` here because you already have `const [user, setUser]` at the top of this file